example/gke/main.tf

```
@@ -59,6 +60,7 @@ module "my-cluster" {
     ]
     tags : {}
     max_pods : 50 # Specific
+    spot_instance : true # to enable preemptible instances, optional
```

I would leave "enable" for stuff that you have to toggle on to be able to use it. Here we are choosing if the instance types are preemptible or not.
```suggestion
    spot_instance : true # create preemptible instances instead of the standard ones, optional
```